code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -1234,8 +1234,8 @@ to_chat will check for valid clients itself already so no need to double check f
 	return hivenumber
 
 /mob/living/carbon/human/get_xeno_hivenumber()
-	if(faction == FACTION_XENO)
-		return XENO_HIVE_NORMAL
+	if(faction == FACTION_ZOMBIE)
+		return FACTION_ZOMBIE
 	return FALSE
 
```

Comment:
uhhh

Reply:
You really want me to refactor factions for xeno? Because that's how it's "properly" done

Reply:
No I mean why are factions being treated like hive numbers here

Reply:
Xeno hive numbers are sub factions for xenos.